### DISCUSSION TITLE
no validation code during auth email and password

### DISCUSSION THREAD

**Discussion by adultviews20** [serverpod/serverpod#1854] [Q&A] [answered]
**Describe the bug**
I try to make just a test auth with standart SignInWithEmailButton,  i did it before many times. success..  but now during creation new user with email and password  validation code doesnt want to come..

![image](https://github.com/serverpod/serverpod/assets/90047188/ce34a215-6b3a-476c-adc0-0f9aa3a6fa01)

![image](https://github.com/serverpod/serverpod/assets/90047188/30c1ddac-fb43-4dde-bd3c-aabed65534e0)

**Serverpod versions**
  serverpod: ^1.1.1
  serverpod_auth_email_flutter: ^1.1.1
  serverpod_auth_server: ^1.1.1

**Platform information**
windows app

Please help

**Comment by yahu1031** [comment]
@adultviews20 any errors from flutter app side?

**Comment by yuriiyudin** [comment]
no

**Comment by yuriiyudin** [comment, marked as answer]
problem fixied... for some reason  my migrations  for auth was not apply.
after apply everything is ok!
